### DISCUSSION TITLE
Load images from google drive

### DISCUSSION THREAD

**Discussion by SlightlyMe** [TheLastBen/fast-stable-diffusion#580] [Q&A]
Is it possible to load the images from google drive? in the colab
Thanks

**Comment by TheLastBen** [comment]
Yes, just enter the path of the folder in the "instance images" cell, the folder should only contain images.

**Reply by SlightlyMe** [reply]
I'll have to figure it out, because I had so much trouble getting dreambooth (see other thread), I uploaded all my photos to my google drive, but I only know how to get a link. I'll look into it, much thanks.

**Reply by SlightlyMe** [reply]
figured it out ::D
